openIt/jsonld-contexts/openIt.jsonld

```diff
@@ -3,16 +3,17 @@
     "Authorization": "https://vocab.egm.io/Authorization",
     "Gate": "https://vocab.egm.io/Gate",
     "Key": "https://vocab.egm.io/Key",
-    "category": "https://schema.org/category",
     "codes": "https://vocab.egm.io/codes",
     "codeValidation": "https://vocab.egm.io/codeValidation",
     "forGate": "https://vocab.egm.io/forGate",
     "givenBy": "https://vocab.egm.io/givenBy",
     "givenTo": "https://vocab.egm.io/givenTo",
     "initDate": "https://vocab.egm.io/initDate",
+    "isAvailableForOrganization": "https://vocab.egm.io/isAvailableForOrganization",
     "keyNumber": "https://vocab.egm.io/keyNumber",
     "name": "https://schema.org/name",
     "opensGate": "https://vocab.egm.io/opensGate",
+    "organization": "https://vocab.egm.io/organization",
```

Comment:
it is a type, not an attribute? if so, it should be Organization.

and this one is better suited: https://schema.org/Organization

Reply:
also, is it a group or an organization?

Reply:
Both I suppose. They used the terminology "Organism". I can swap to Organization entities. Would be better to have them separate anyways for managing them (adding, deleting, updating)